systemtest/src/test/java/io/strimzi/systemtest/operators/topic/ThrottlingQuotaST.java

```diff
@@ -0,0 +1,389 @@
+        String createPodName;
+        int offset = 100;
+        int iterations = topicsCountOverQuota / 100;
+        for (int i = 0; i < iterations; i++) {
+            LOGGER.info("Executing {}/{} iteration.", i + 1, iterations);
+            createAdminClientJob = new KafkaAdminClient.Builder()
+                .withAdminName(createAdminName)
+                .withBootstrapAddress(KafkaResources.plainBootstrapAddress(CLUSTER_NAME))
+                .withTopicName(topicNamePrefix)
+                .withTopicCount(100)
+                .withTopicOffset(i * offset)
+                .withNamespaceName(NAMESPACE)
+                .withTopicOperation(AdminClientOperations.CREATE_TOPICS.toString())
+                .withAdditionalConfig(getAdminClientConfig(kafkaUsername))
+                .build();
+            resourceManager.createResource(extensionContext, true, createAdminClientJob.defaultAdmin().build());
+            ClientUtils.waitForClientSuccess(createAdminName, NAMESPACE, 10);
+            createPodName = kubeClient().listPods("job-name", createAdminName).get(0).getMetadata().getName();
+            PodUtils.waitUntilMessageIsInPodLogs(createPodName, "All topics created", Duration.ofMinutes(6).toMillis());
+            JobUtils.deleteJobWithWait(NAMESPACE, createAdminName);
+        }
+
+        // Test delete all topics at once - should fail on Throttling Quota limit
+        KafkaAdminClient deleteAdminClientJob = new KafkaAdminClient.Builder()
+                .withAdminName(deleteAdminName)
+                .withBootstrapAddress(KafkaResources.plainBootstrapAddress(CLUSTER_NAME))
+                .withTopicName(topicNamePrefix)
+                .withTopicCount(topicsCountOverQuota)
+                .withNamespaceName(NAMESPACE)
+                .withTopicOperation(AdminClientOperations.DELETE_TOPICS.toString())
+                .withAdditionalConfig(getAdminClientConfig(kafkaUsername))
+                .build();
+        resourceManager.createResource(extensionContext, deleteAdminClientJob.defaultAdmin().build());
+        String deletePodName = kubeClient().listPods("job-name", deleteAdminName).get(0).getMetadata().getName();
+        PodUtils.waitUntilMessageIsInPodLogs(
+                deletePodName,
+                "org.apache.kafka.common.errors.ThrottlingQuotaExceededException: The throttling quota has been exceeded.",
+                Duration.ofMinutes(5).toMillis()
+        );
+        JobUtils.deleteJobWithWait(NAMESPACE, deleteAdminName);
+
+        // Teardown - delete all (remaining) topics (within Quota limits) in multiple rounds using starting offsets
+        for (int i = 0; i < iterations; i++) {
+            LOGGER.info("Executing {}/{} iteration for {}.", i + 1, iterations, deleteAdminName);
+            deleteAdminClientJob = deleteAdminClientJob.toBuilder()
+                    .withTopicOffset(i * offset)
+                    .withTopicCount(100)
+                    .withAdditionalConfig("")
```

Comment:
Why do we need to set this for every iteration? When you add additional config for same user on line 138, is there any difference on line 155? Or why do you set this to `""` here?